includes/functions-html.php

```diff
@@ -757,7 +760,7 @@ function yourls_html_menu() {
 	$admin_sublinks = yourls_apply_filter( 'admin_sublinks', $admin_sublinks );
 	
 	// Now output menu
-	echo '<ul id="admin_menu">'."\n";
+	echo '<nav role="navigation"><ul id="admin_menu">'."\n";
```

Comment:
Well, HTML elements are semantic :-) 
`footer` means that it's a footer. A div means nothing. You can see it is a footer because ... well you can see. Someone who can't see, doesn't know it is a footer. So landamark say its a block with content info, and html5 `footer` say its a footer :-D 
Do i convince you ? 

Even if HTML5 elements are not well supported in screen reader today, they will, and it's not that bad to use the new HTML5, the new W3C Recommendation ;-)

Semantic is good for the web :-)